src/ethereum/fees/feeProviders.ts

```diff
@@ -62,6 +68,59 @@ export const FeeProviders = (
   }
 }
 
+export const fetchFeesFromRpc = async (
+  fetch: EdgeFetchFunction,
+  currencyInfo: EdgeCurrencyInfo,
+  initOptions: EthereumInitOptions,
+  log: EdgeLog,
+  networkInfo: EthereumNetworkInfo
+): Promise<EthereumBaseMultiplier | undefined> => {
+  const { rpcServers, supportsEIP1559 = false } = networkInfo
+  if (supportsEIP1559) return
+
+  const server = pickRandom(rpcServers, 1)[0]
+
+  const opts = {
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    },
+    method: 'POST',
+    body: JSON.stringify({
+      method: 'eth_gasPrice',
+      params: [],
+      id: 1,
+      jsonrpc: '2.0'
+    })
+  }
+
+  const fetchResponse = await fetch(server, opts)
+  if (!fetchResponse.ok) {
+    const text = await fetchResponse.text()
+    throw new Error(`fetchFeesFromRpc fetch error: ${text}`)
+  }
+
+  const json = await fetchResponse.json()
+  const rpcGasResponse = asMaybe(asRpcResultString)(json)
+
+  if (rpcGasResponse == null) {
+    throw new Error(`fetchFeesFromRpc ${server} returned invalid json: ${json}`)
+  }
+
+  const { result } = rpcGasResponse
+  const gasPrice = hexToDecimal(result)
+
+  const out = {
+    lowFee: mul(gasPrice, '1'),
+    standardFeeLow: mul(gasPrice, '1.06'),
+    standardFeeHigh: mul(gasPrice, '1.12'),
+    highFee: mul(gasPrice, '1.25')
+  }
+  log.warn(`fetchFeesFromRpc: ${currencyInfo.currencyCode}`)
```

Comment:
Nothing is wrong here, so we should use `log()`, not `log.warn()`

Reply:
Went ahead and quieted the other fee fetching functions as well